### PR TITLE
Added test case for minikube config unset

### PR DESCRIPTION
**Pull request by hpandeycodeit**
Added Test Case for `minikube config unset`.

### CODE REVIEW

File: cmd/minikube/cmd/config/unset_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import "testing"
+
+func TestUnsetConfig(t *testing.T) {
+	err := Set("dashboard", "true")
```
Comment (tstromberg) [inline cmd/minikube/cmd/config/unset_test.go]: Make sure to check this err too.

File: cmd/minikube/cmd/config/unset_test.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	err = unset("dashboard")
+	if err != nil {
+		t.Fatalf("Failed to unset property ")
```
Comment (tstromberg) [inline cmd/minikube/cmd/config/unset_test.go]: Prefer t.Errorf over t.Fatalf whenever possible. That way you can see more failed test information without rerunning it later. In this particular case, it doesn't change the behavior.

Comment (hpandeycodeit) [reply cmd/minikube/cmd/config/unset_test.go]: Thanks for pointing it out. I have updated it.

File: cmd/minikube/cmd/config/unset_test.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+package config
+
+import "testing"
+
+func TestUnsetConfig(t *testing.T) {
+	propName := "dashboard"
+	propValue := "true"
+	err := Set(propName, propValue)
+	if err != nil {
+		t.Errorf("Failed to set the propert %q", propName)
```
[the remark this answers is not in the document]
Comment (hpandeycodeit) [reply cmd/minikube/cmd/config/unset_test.go]: done

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by tstromberg** [comment]
Test failed with:
```
E0516 20:45:27.894418   15408 console.go:120] no error file has been set
FAIL	k8s.io/minikube/cmd/minikube/cmd/config	0.032s
```

The error mentioned is new, but shouldn't be fatal. It's curious that the actual name of the failing test isn't being displayed though? I wonder if this is unintended fall-out from #4253

Do you mind merging your PR from head? I'm curious if `make test` fails for you.

**Comment by hpandeycodeit** [comment]
@tstromberg  It's working fine in my local even after merging changes from the HEAD. 

```

= go test ===============================================================
ok  	k8s.io/minikube/cmd/minikube/cmd	0.436s	coverage: 17.5% of statements
ok  	k8s.io/minikube/cmd/minikube/cmd/config	1.296s	coverage: 25.9% of statements
ok  	k8s.io/minikube/cmd/util	0.077s	coverage: 7.1% of statements
ok  	k8s.io/minikube/pkg/drivers	0.100s	coverage: 19.2% of statements
ok  	k8s.io/minikube/pkg/drivers/hyperkit	0.319s	coverage: 15.6% of statements
ok  	k8s.io/minikube/pkg/minikube/assets	0.171s	coverage: 61.9% of statements
ok  	k8s.io/minikube/pkg/minikube/bootstrapper	3.020s	coverage: 21.9% of statements
ok  	k8s.io/minikube/pkg/minikube/bootstrapper/kubeadm	0.413s	coverage: 30.5% of statements
ok  	k8s.io/minikube/pkg/minikube/cluster	0.796s	coverage: 52.6% of statements
ok  	k8s.io/minikube/pkg/minikube/config	0.204s	coverage: 17.6% of statements
ok  	k8s.io/minikube/pkg/minikube/console	0.047s	coverage: 70.7% of statements
ok  	k8s.io/minikube/pkg/minikube/cruntime	0.044s	coverage: 63.8% of statements
ok  	k8s.io/minikube/pkg/minikube/logs	0.147s	coverage: 1.5% of statements
ok  	k8s.io/minikube/pkg/minikube/machine	0.172s	coverage: 12.0% of statements
ok  	k8s.io/minikube/pkg/minikube/notify	0.152s	coverage: 76.9% of statements
ok  	k8s.io/minikube/pkg/minikube/problem	0.072s	coverage: 42.9% of statements
ok  	k8s.io/minikube/pkg/minikube/registry	0.123s	coverage: 81.8% of statements
ok  	k8s.io/minikube/pkg/minikube/service	0.512s	coverage: 41.0% of statements
ok  	k8s.io/minikube/pkg/minikube/sshutil	0.397s	coverage: 75.0% of statements
ok  	k8s.io/minikube/pkg/minikube/tunnel	2.193s	coverage: 59.0% of statements
ok  	k8s.io/minikube/pkg/util	1.268s	coverage: 50.4% of statements

```

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by hpandeycodeit** [comment]
@minikube-bot OK to test

**Comment by tstromberg** [comment]
Thank you!

**Comment by hpandeycodeit** [comment]
Thank you @tstromberg! :)

**Pull request merged**